test/Makefile

```
@@ -0,0 +1,47 @@
+BUILD_DATE ?= $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+# LDFLAGS with version info
+LDFLAGS += -X github.com/openshift/ingress-node-firewall/test/version.Version=$(VERSION)
+LDFLAGS += -X github.com/openshift/ingress-node-firewall/test/version.GitCommit=$(GIT_COMMIT)
+LDFLAGS += -X github.com/openshift/ingress-node-firewall/test/version.BuildDate=$(BUILD_DATE)
+
+.PHONY: all
+all: build-e2e-tests
+
+.PHONY: build-e2e-tests
+build-e2e-tests: ## Build the extended e2e test binary (static, ART compliant)
```

checkmake (0.3.2)

[warning] Target body for "build-e2e-tests" exceeds allowed length of 5 lines (7).

(maxbodylength)